client/me/billing-history/receipt.jsx

```diff
@@ -138,6 +139,7 @@ const BillingReceipt = React.createClass( {
 		return (
 			<div>
 				<DocumentHead title={ translate( 'Billing History' ) } />
+				<PageViewTracker path="/me/purchases/billing/:receiptId" title="Me > Billing History > Receipt" />
```

Comment:
Did we mean to change the path here? In this diff it'll be `/me/purchases/billing/:receiptId` instead of `/me/purchases/billing/receipt`